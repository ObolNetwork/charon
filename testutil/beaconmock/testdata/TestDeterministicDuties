[
 {
  "pubkey": "0x8dae41352b69f2b3a1c0b05330c1bf65f03730c520273028864b11fcb94d8ce8f26d64f979a0ee3025467f45fd2241ea",
  "slot": "32",
  "validator_index": "2",
  "committee_index": "0",
  "committee_length": "32",
  "committees_at_slot": "32",
  "validator_committee_index": "2"
 }
]