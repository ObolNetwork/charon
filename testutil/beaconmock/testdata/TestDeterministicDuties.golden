[
 {
  "pubkey": "0x8ee91545183c8c2db86633626f5074fd8ef93c4c9b7a2879ad1768f600c5b5906c3af20d47de42c3b032956fa8db1a76",
  "slot": "16",
  "validator_index": "2",
  "committee_index": "0",
  "committee_length": "16",
  "committees_at_slot": "16",
  "validator_committee_index": "2"
 }
]